src/pyhf/workspace.py

```diff
@@ -10,11 +17,10 @@
 
 
 class Workspace(_ChannelSummaryMixin, dict):
-    """
-    A JSON-serializable object that is built from an object that follows the `workspace.json` schema.
-    """
+    """A JSON-serializable object that is built from an object that follows the `workspace.json` schema."""
```

Comment:
Can we use the ~ syntax to link this?

Reply:
No, we can't. @kratsg Can you think of a better way to link this then the following?

```diff
diff --git a/src/pyhf/workspace.py b/src/pyhf/workspace.py
index ece88a9..34193f9 100644
--- a/src/pyhf/workspace.py
+++ b/src/pyhf/workspace.py
@@ -17,7 +17,11 @@ log = logging.getLogger(__name__)
 
 
 class Workspace(_ChannelSummaryMixin, dict):
-    """A JSON-serializable object that is built from an object that follows the `workspace.json` schema."""
+    """
+    A JSON-serializable object that is built from an object that follows the :obj:`workspace.json` schema_.
+
+    .. _schema: https://scikit-hep.org/pyhf/likelihood.html#workspace
+    """
 
     def __init__(self, spec, **config_kwargs):
         """Workspaces hold the model, data and measurements."""
```

Reply:
Though this does introduce the following warning when running `make html` for the docs:

```
/home/feickert/Code/GitHub/pyhf/docs/api.rst:47:<autosummary>:1: WARNING: Unknown target name: "schema".
```

So that should get corrected.

Reply:
Fixed by 572102db1353a7e09f66485907e50a41d7d6bf17